src/internal/interpreter/variables.ts

```diff
@@ -109,6 +109,19 @@ export const getValue = (
   return result;
 };
 
+/**
+ * Stringifies a primitive variable.
+*/
+export const variableToString = (
```

Comment:
suggestion: It may not be clear what exactly this does because the comment says primitive, but it can take anything. Could we either make it only take `string | boolean | number` or explain it in the comment?

Reply:
You are right, the comment is wrong. I'm not sure why I wrote "primitive".

Reply:
Should we try to handle objects/arrays specially? It doesn't seem to be a problem right now but maybe we are just missing a bug here.

Reply:
I'm not sure. It worked like this so far, so let's leave it until we have a case for changing it :shrug: